### PR TITLE
DTSPO-6258 - dev: migrate jd to fluxv2

### PR DESCRIPTION
**Pull request by endakelly**
- add jd to fluxv2 dev
- dev: migrate jd to flux v2

### CODE REVIEW

File: apps/juror-digital/identity/juror-digital-azure-identity.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+apiVersion: "aadpodidentity.k8s.io/v1"
+kind: AzureIdentity
+metadata:
+  name: juror-digital
+spec:
+  type: 0
+  resourceID: replace
```
Comment (timja) [inline apps/juror-digital/identity/juror-digital-azure-identity.yaml:7]: where's the dev identity for this?

Comment (endakelly) [reply apps/juror-digital/identity/juror-digital-azure-identity.yaml:7]: There wasn't one in the flux v1 setup but I've added it there

Comment (timja) [reply apps/juror-digital/identity/juror-digital-azure-identity.yaml:8]: Must’ve been somewhere

**Review by timja — APPROVED** [review]

**Comment by timja** [comment]
Conflicts

**Pull request merged**